### ISSUE TITLE
Reconnect while rescanning does not continue rescanning.

### ISSUE THREAD

**Issue by Christewart** [bitcoin-s/bitcoin-s#5231]
Observed on 930382dd

While rescanning the wallet, if the node reconnects for whatever reason, it appears that are our rescan does not resume when the reconnection occurs. 

```
2023-09-08 00:11:01,134UTC INFO [DLCWallet$DLCWalletImpl] Found 4 matches from start=Some(724962) to end=Some(725561)
2023-09-08 00:11:01,134UTC INFO [DLCWallet$DLCWalletImpl] Searching filters from start=726162 to end=726761
2023-09-08 00:28:55,732UTC INFO [DLCWallet$DLCWalletImpl] Found a match in block 725805
2023-09-08 00:28:56,085UTC INFO [DLCWallet$DLCWalletImpl] Found a match in block 725591
2023-09-08 00:28:56,197UTC INFO [DLCWallet$DLCWalletImpl] Found a match in block 725593
2023-09-08 00:28:56,354UTC INFO [DLCWallet$DLCWalletImpl] Found a match in block 725598
2023-09-08 00:28:57,059UTC INFO [DLCWallet$DLCWalletImpl] Found a match in block 725933
2023-09-08 00:28:57,251UTC INFO [DLCWallet$DLCWalletImpl] Requesting 5 block(s)
2023-09-08 00:45:45,392UTC INFO [PeerManager] Disconnected peer=Peer(neutrino.suredbits.com:8333) peers=Set(Peer(neutrino.suredbits.com:8333)) state=DoneSyncing(Set(Peer(neutrino.suredbits.com:8333))) forceReconnect=false finder.isDefined=true peerDataMap=List(Peer(neutrino.suredbits.com:8333))
2023-09-08 00:45:45,393UTC INFO [PeerFinder] Attempting to reconnect peer=Peer(neutrino.suredbits.com:8333)
2023-09-08 00:45:45,406UTC INFO [PeerMessageSender] Attempting to reconnect peer=Peer(neutrino.suredbits.com:8333)
2023-09-08 00:45:45,921UTC INFO [PeerMessageSender] Attempting to connect to peer=Peer(neutrino.suredbits.com:8333)
2023-09-08 00:45:45,924UTC INFO [PeerMessageSender] Connected to remote=localhost/127.0.0.1:61941  local=/127.0.0.1:61127
2023-09-08 01:03:37,242UTC INFO [PeerMessageSender] Connection with peer=Peer(neutrino.suredbits.com:8333) failed with err=Tor connection request failed to target=neutrino.suredbits.com/<unresolved>:8333 errMsg=java.lang.RuntimeException: TTLExpired
2023-09-08 01:03:37,242UTC INFO [PeerManager] Disconnected peer=Peer(neutrino.suredbits.com:8333) peers=Set() state=DoneSyncing(Set()) forceReconnect=false finder.isDefined=true peerDataMap=List()
2023-09-08 01:03:37,242UTC INFO [PeerMessageSender] Disconnecting peer=Peer(neutrino.suredbits.com:8333)
2023-09-08 01:19:49,457UTC WARN [HikariPool] bitcoin-s-chain-db - Thread starvation or clock leap detected (housekeeper delta=1h9m411ms).
2023-09-08 01:19:49,457UTC WARN [HikariPool] bitcoin-s.dlc.db - Thread starvation or clock leap detected (housekeeper delta=1h9m411ms).
2023-09-08 01:19:49,708UTC WARN [HikariPool] bitcoin-s-node-db - Thread starvation or clock leap detected (housekeeper delta=1h9m412ms).
2023-09-08 01:19:49,851UTC WARN [HikariPool] bitcoin-s.dlc.db - Thread starvation or clock leap detected (housekeeper delta=1h9m415ms).
2023-09-08 01:19:49,865UTC WARN [HikariPool] bitcoin-s-wallet-db - Thread starvation or clock leap detected (housekeeper delta=1h9m416ms).
2023-09-08 01:36:54,770UTC ERROR [Socks5ProxyGraphStage] Failed to connect mempool.space:443 via tor
java.lang.RuntimeException: TTLExpired
	at org.bitcoins.tor.TorConnectionError.exn(TorConnectionError.scala:6)
	at org.bitcoins.tor.TorConnectionError.exn$(TorConnectionError.scala:6)
	at org.bitcoins.tor.TorConnectionError$TTLExpired$.exn(TorConnectionError.scala:75)
	at org.bitcoins.tor.Socks5Connection$.tryParseConnectedAddress(Socks5Connection.scala:212)
	at org.bitcoins.tor.Socks5ProxyGraphStage$$anon$1.org$bitcoins$tor$Socks5ProxyGraphStage$$anon$$parseResponse(Socks5ClientTransport.scala:200)
	at org.bitcoins.tor.Socks5ProxyGraphStage$$anon$1$$anon$2.onPush(Socks5ClientTransport.scala:226)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:542)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:423)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:650)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:521)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:625)
	at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:800)
	at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:818)
	at akka.actor.Actor.aroundReceive(Actor.scala:537)
	at akka.actor.Actor.aroundReceive$(Actor.scala:535)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:716)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:579)
	at akka.actor.ActorCell.invoke(ActorCell.scala:547)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
	at akka.dispatch.Mailbox.run(Mailbox.scala:231)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
2023-09-08 10:34:59,965UTC INFO [PeerManager] Running inactivity checks for peers=List()
2023-09-08 10:34:59,965UTC INFO [PeerManager] Stopping PeerManager
2023-09-08 10:34:59,965UTC INFO [PeerFinder] Stopping PeerFinder
2023-09-08 10:34:59,966UTC INFO [PeerFinder] Done stopping PeerFinder
2023-09-08 10:34:59,966UTC INFO [PeerManager] Stopped PeerManager. Took 1 ms 
2023-09-08 10:34:59,982UTC INFO [PeerMessageSender] Attempting to connect to peer=Peer(neutrino.suredbits.com:8333)
2023-09-08 10:34:59,984UTC INFO [PeerMessageSender] Connected to remote=localhost/127.0.0.1:61941  local=/127.0.0.1:61723
2023-09-08 10:35:00,545UTC INFO [Socks5Connection$] Tor connection request succeeded. target=neutrino.suredbits.com/<unresolved>:8333 connectedAddress=/0.0.0.0:0
2023-09-08 10:35:00,795UTC INFO [PeerManager] Disconnected peer=Peer(neutrino.suredbits.com:8333) peers=Set() state=DoneSyncing(Set()) forceReconnect=false finder.isDefined=true peerDataMap=List()
2023-09-08 10:35:00,795UTC INFO [PeerMessageSender] Disconnecting peer=Peer(neutrino.suredbits.com:8333)
2023-09-08 10:35:32,420UTC INFO [PeerFinder] Done starting PeerFinder
2023-09-08 10:35:32,420UTC INFO [PeerManager] Done starting PeerManager
2023-09-08 10:40:00,016UTC INFO [PeerManager] Running inactivity checks for peers=List()
2023-09-08 10:40:00,016UTC INFO [PeerManager] Stopping PeerManager
2023-09-08 10:40:00,016UTC INFO [PeerFinder] Stopping PeerFinder
2023-09-08 10:40:00,017UTC INFO [PeerFinder] Done stopping PeerFinder
2023-09-08 10:40:00,018UTC INFO [PeerManager] Stopped PeerManager. Took 2 ms 
2023-09-08 10:40:00,035UTC INFO [PeerMessageSender] Attempting to connect to peer=Peer(neutrino.suredbits.com:8333)
2023-09-08 10:40:00,038UTC INFO [PeerMessageSender] Connected to remote=localhost/127.0.0.1:61941  local=/127.0.0.1:62034
2023-09-08 10:40:00,437UTC INFO [Socks5Connection$] Tor connection request succeeded. target=neutrino.suredbits.com/<unresolved>:8333 connectedAddress=/0.0.0.0:0
2023-09-08 10:40:00,676UTC INFO [PeerManager] Disconnected peer=Peer(neutrino.suredbits.com:8333) peers=Set() state=DoneSyncing(Set()) forceReconnect=false finder.isDefined=true peerDataMap=List()
2023-09-08 10:40:00,676UTC INFO [PeerMessageSender] Disconnecting peer=Peer(neutrino.suredbits.com:8333)
2023-09-08 10:40:30,571UTC INFO [PeerFinder] Done starting PeerFinder
2023-09-08 10:40:30,571UTC INFO [PeerManager] Done starting PeerManager
2023-09-08 11:17:10,828UTC WARN [HikariPool] bitcoin-s.dlc.db - Thread starvation or clock leap detected (housekeeper delta=32m43s153ms).
2023-09-08 11:17:10,842UTC WARN [HikariPool] bitcoin-s-chain-db - Thread starvation or clock leap detected (housekeeper delta=32m43s159ms).
2023-09-08 11:17:11,053UTC WARN [HikariPool] bitcoin-s-node-db - Thread starvation or clock leap detected (housekeeper delta=32m43s151ms).
2023-09-08 11:17:11,204UTC WARN [HikariPool] bitcoin-s.dlc.db - Thread starvation or clock leap detected (housekeeper delta=32m43s152ms).
2023-09-08 11:17:11,226UTC WARN [HikariPool] bitcoin-s-wallet-db - Thread starvation or clock leap detected (housekeeper delta=32m43s153ms).
2023-09-08 11:17:13,202UTC INFO [PeerManager] Running inactivity checks for peers=List()
2023-09-08 11:17:13,203UTC INFO [PeerManager] Stopping PeerManager
2023-09-08 11:17:13,203UTC INFO [PeerFinder] Stopping PeerFinder
2023-09-08 11:17:13,204UTC INFO [PeerFinder] Done stopping PeerFinder
2023-09-08 11:17:13,206UTC INFO [PeerManager] Stopped PeerManager. Took 3 ms 
2023-09-08 11:17:13,239UTC INFO [PeerMessageSender] Attempting to connect to peer=Peer(neutrino.suredbits.com:8333)
2023-09-08 11:17:13,241UTC INFO [PeerMessageSender] Connected to remote=localhost/127.0.0.1:61941  local=/127.0.0.1:62186
2023-09-08 11:17:13,261UTC INFO [PeerFinder] Done starting PeerFinder
2023-09-08 11:17:13,261UTC INFO [PeerManager] Done starting PeerManager
2023-09-08 11:35:07,451UTC ERROR [Socks5ProxyGraphStage] Failed to connect mempool.space:443 via tor
java.lang.RuntimeException: TTLExpired
	at org.bitcoins.tor.TorConnectionError.exn(TorConnectionError.scala:6)
	at org.bitcoins.tor.TorConnectionError.exn$(TorConnectionError.scala:6)
	at org.bitcoins.tor.TorConnectionError$TTLExpired$.exn(TorConnectionError.scala:75)
	at org.bitcoins.tor.Socks5Connection$.tryParseConnectedAddress(Socks5Connection.scala:212)
	at org.bitcoins.tor.Socks5ProxyGraphStage$$anon$1.org$bitcoins$tor$Socks5ProxyGraphStage$$anon$$parseResponse(Socks5ClientTransport.scala:200)
	at org.bitcoins.tor.Socks5ProxyGraphStage$$anon$1$$anon$2.onPush(Socks5ClientTransport.scala:226)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:542)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:423)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:650)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:521)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:625)
	at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:800)
	at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:818)
	at akka.actor.Actor.aroundReceive(Actor.scala:537)
	at akka.actor.Actor.aroundReceive$(Actor.scala:535)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:716)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:579)
	at akka.actor.ActorCell.invoke(ActorCell.scala:547)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
	at akka.dispatch.Mailbox.run(Mailbox.scala:231)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
2023-09-08 15:22:27,698UTC INFO [PeerManager] Running inactivity checks for peers=List()
2023-09-08 15:22:27,699UTC INFO [PeerManager] Stopping PeerManager
2023-09-08 15:22:27,699UTC INFO [PeerFinder] Stopping PeerFinder
2023-09-08 15:22:27,702UTC INFO [PeerFinder] Done stopping PeerFinder
2023-09-08 15:22:27,704UTC INFO [PeerManager] Stopped PeerManager. Took 5 ms 
2023-09-08 15:22:27,736UTC INFO [PeerMessageSender] Attempting to connect to peer=Peer(neutrino.suredbits.com:8333)
2023-09-08 15:22:27,740UTC INFO [PeerMessageSender] Connected to remote=localhost/127.0.0.1:61941  local=/127.0.0.1:63448
2023-09-08 15:22:28,817UTC INFO [Socks5Connection$] Tor connection request succeeded. target=neutrino.suredbits.com/<unresolved>:8333 connectedAddress=/0.0.0.0:0
2023-09-08 15:22:29,324UTC INFO [PeerManager] Connected to peer Peer(neutrino.suredbits.com:8333) with filters. Connected peer count 1
2023-09-08 15:22:29,390UTC INFO [PeerManager] Starting sync node, height=806667 hash=0000000000000000000052ed575b91612216b217b9acc9853885dc3d1fd61ef1 filterHeaderCount=806667 filterCount=806667 peerOpt=Some(Peer(neutrino.suredbits.com:8333))
2023-09-08 15:22:29,936UTC INFO [DataMessageHandler] Received headers message with 87 headers from peer=Peer(neutrino.suredbits.com:8333) state=DoneSyncing(Set(Peer(neutrino.suredbits.com:8333)))
2023-09-08 15:22:30,009UTC INFO [ChainHandler] Processed headers from height=806667 to 806754. Best hash=0000000000000000000348795a1d4018654ebfea82473139e5666430c5603b5d
2023-09-08 15:22:30,009UTC INFO [PeerManager$] Now syncing filter headers from Peer(neutrino.suredbits.com:8333) in state=HeaderSync(Peer(neutrino.suredbits.com:8333),Set(Peer(neutrino.suredbits.com:8333)))
2023-09-08 15:22:30,786UTC INFO [ChainHandler] Processed filters headers from height=806668 to 806754. Best filterheader.blockHash=0000000000000000000348795a1d4018654ebfea82473139e5666430c5603b5d
2023-09-08 15:22:31,521UTC INFO [DataMessageHandler] Beginning to sync filters from startHeight=806667
2023-09-08 15:22:31,549UTC INFO [PeerManager$] Requesting compact filters from FilterSyncMarker(startHeight=806668, stopBlockHash=0000000000000000000348795a1d4018654ebfea82473139e5666430c5603b5d) with peer=Peer(neutrino.suredbits.com:8333)
2023-09-08 15:22:34,484UTC INFO [ChainHandler] Processed filters from height=806668 to 806754. Best filter.blockHash=0000000000000000000348795a1d4018654ebfea82473139e5666430c5603b5d
2023-09-08 15:22:34,485UTC INFO [DataMessageHandler] We are synced
2023-09-08 15:22:34,493UTC INFO [DLCWallet$DLCWalletImpl] Processing 87 block filters for 2 utxos and 1200 scripts
2023-09-08 15:23:00,303UTC INFO [PeerFinder] Done starting PeerFinder
2023-09-08 15:23:00,303UTC INFO [PeerManager] Done starting PeerManager
2023-09-08 15:27:27,743UTC INFO [PeerManager] Running inactivity checks for peers=List(Peer(neutrino.suredbits.com:8333))

```

**Comment by Christewart** [comment]
Another instance on 930382dd

```
2023-09-08 15:34:52,698UTC INFO [DLCWallet$DLCWalletImpl] Starting Wallet
2023-09-08 15:34:52,712UTC INFO [WalletAppConfig] Starting wallet rebroadcast task
2023-09-08 15:34:52,759UTC INFO [DLCWallet$DLCWalletImpl] Starting rescanning the wallet= from None to None useCreationTime=true
2023-09-08 15:34:53,858UTC INFO [TorAppConfig] Tor daemon is fully started, it took=2097ms
2023-09-08 15:34:53,859UTC INFO [NodeAppConfig] Initializing node setup
2023-09-08 15:34:53,862UTC INFO [DbValidate] Successfully validated 5 migrations (execution time 00:00.001s)
2023-09-08 15:34:53,863UTC INFO [DbMigrate] Current version of schema "main": 5
2023-09-08 15:34:53,863UTC INFO [DbMigrate] Schema "main" is up to date. No migration necessary.
2023-09-08 15:34:53,863UTC INFO [NodeAppConfig] Applied org.flywaydb.core.api.output.MigrateResult@533dac9 migrations fro the node project
2023-09-08 15:34:58,100UTC INFO [DLCWallet$DLCWalletImpl] Beginning to search for matches between 681761:806754
2023-09-08 15:34:58,118UTC INFO [DLCWallet$DLCWalletImpl] Searching filters from start=681761 to end=681761
2023-09-08 15:34:58,120UTC INFO [DLCWalletNeutrinoBackendLoader] Done loading wallet=None
2023-09-08 15:34:58,127UTC INFO [DLCWallet$DLCWalletImpl] Searching filters from start=681762 to end=682361
2023-09-08 15:34:58,131UTC WARN [Server] RPC password is not set (rpchost=127.0.0.1)
2023-09-08 15:34:58,266UTC INFO [BitcoinSServerMain] Done configuring node, it took=5832ms
2023-09-08 15:34:58,267UTC INFO [Server] Started Bitcoin-S websocket at /127.0.0.1:19999
2023-09-08 15:34:58,267UTC INFO [Server] Started Bitcoin-S HTTP server at /127.0.0.1:9999
2023-09-08 15:34:58,268UTC INFO [BitcoinSServerMain] Starting neutrino node sync, it took=5834ms
2023-09-08 15:34:58,284UTC INFO [NeutrinoNode] Starting node
2023-09-08 15:34:58,333UTC INFO [DLCNode] Binding server to /0.0.0.0:2862, with tor hidden service: true
2023-09-08 15:34:58,357UTC INFO [TorProtocolHandler] Tor version 0.4.7.10
2023-09-08 15:34:58,433UTC INFO [TorController$] Created hidden service with address=ugecp2d5iqv5qdohptqt43sbyq4lutsrl2yixnnyxxjbtcalj7hwx4qd.onion/<unresolved>:2862
2023-09-08 15:34:58,441UTC INFO [DLCServer] Bound at /[0:0:0:0:0:0:0:0]:2862
2023-09-08 15:34:58,452UTC INFO [PeerMessageSender] Attempting to connect to peer=Peer(neutrino.suredbits.com:8333)
2023-09-08 15:34:58,457UTC INFO [PeerMessageSender] Connected to remote=localhost/127.0.0.1:28059  local=/127.0.0.1:64430
2023-09-08 15:34:58,737UTC INFO [DLCWallet$DLCWalletImpl] Requesting 0 block(s)
2023-09-08 15:34:58,737UTC INFO [DLCWallet$DLCWalletImpl] Found 0 matches from start=Some(681761) to end=Some(681761)
2023-09-08 15:34:58,737UTC INFO [DLCWallet$DLCWalletImpl] Searching filters from start=682362 to end=682961
2023-09-08 15:34:59,032UTC INFO [Socks5Connection$] Tor connection request succeeded. target=neutrino.suredbits.com/<unresolved>:8333 connectedAddress=/0.0.0.0:0
2023-09-08 15:34:59,310UTC INFO [DLCWallet$DLCWalletImpl] Found a match in block 681989
2023-09-08 15:34:59,318UTC INFO [PeerManager] Disconnected peer=Peer(neutrino.suredbits.com:8333) peers=Set() state=DoneSyncing(Set()) forceReconnect=false finder.isDefined=true peerDataMap=List()
2023-09-08 15:34:59,319UTC INFO [PeerMessageSender] Disconnecting peer=Peer(neutrino.suredbits.com:8333)
2023-09-08 15:35:00,815UTC INFO [DLCWallet$DLCWalletImpl] Found a match in block 682252
2023-09-08 15:35:00,920UTC INFO [DLCWallet$DLCWalletImpl] Found a match in block 681892
2023-09-08 15:35:01,174UTC INFO [DLCWallet$DLCWalletImpl] Found a match in block 682117
2023-09-08 15:35:01,496UTC INFO [DLCWallet$DLCWalletImpl] Requesting 4 block(s)
2023-09-08 15:35:01,501UTC WARN [PeerManager] Cannot find peerOpt=None in set of peers=Set() to send message=getdata to. It may have been disconnected, or we have no peers.
2023-09-08 15:35:19,325UTC INFO [BitcoinSServerMain$] @@@@@@@@@@@@@@@@@@@@@ Shutting down BitcoinSServerMain$ @@@@@@@@@@@@@@@@@@@@@
2023-09-08 15:35:19,326UTC ERROR [BitcoinSServerMain] Exiting process
2023-09-08 15:35:19,350UTC INFO [NodeCallbackStreamManager] Done draining akka streams for NodeCallbackStreamManager, it took=22ms
2023-09-08 15:35:19,352UTC INFO [HikariDataSource] bitcoin-s-node-db - Shutdown initiated...
2023-09-08 15:35:19,355UTC INFO [HikariDataSource] bitcoin-s-node-db - Shutdown completed.
2023-09-08 15:35:19,357UTC INFO [HikariDataSource] bitcoin-s.dlc.db - Shutdown initiated...
2023-09-08 15:35:19,359UTC INFO [HikariDataSource] bitcoin-s.dlc.db - Shutdown completed.
2023-09-08 15:35:19,362UTC INFO [HikariDataSource] bitcoin-s-chain-db - Shutdown initiated...
2023-09-08 15:35:19,366UTC INFO [HikariDataSource] bitcoin-s-chain-db - Shutdown completed.
2023-09-08 15:35:19,368UTC INFO [TorClient] No process found, binary wasn't started!
2023-09-08 15:35:19,373UTC INFO [HikariDataSource] bitcoin-s.dlc.db - Shutdown initiated...
2023-09-08 15:35:19,374UTC INFO [HikariDataSource] bitcoin-s.dlc.db - Shutdown completed.
2023-09-08 15:35:19,374UTC INFO [WalletAppConfig] Stopping wallet rebroadcast task
2023-09-08 15:35:19,375UTC INFO [HikariDataSource] bitcoin-s-wallet-db - Shutdown initiated...
2023-09-08 15:35:19,378UTC INFO [HikariDataSource] bitcoin-s-wallet-db - Shutdown completed.
2023-09-08 15:35:19,378UTC INFO [DLCWallet$DLCWalletImpl] Rescan terminated early, don't reset isRescanning flag as new wallet is likely being loaded
2023-09-08 15:35:19,378UTC INFO [DLCWalletNeutrinoBackendLoader] DLCWalletLoaderApi stopped
2023-09-08 15:35:19,383UTC INFO [ServerBindings] ServerBindings stopped
2023-09-08 15:35:19,384UTC INFO [BitcoinSServerMain] Stopped neutrino node
```

**Comment by Christewart** [comment]
fixed in #5800
